packages/web/src/components/common/select-token/SelectToken.tsx

```diff
@@ -23,8 +26,10 @@ const SelectToken: React.FC<SelectTokenProps> = ({
   changeKeyword,
   changeToken,
   close,
+  callback,
 }) => {
-
+  const themeKey = useAtomValue(ThemeState.themeKey);
+  useEscCloseModal(close);
```

Comment:
How about using this hook in a container?

Reply:
updated